src/blocks/icon/deprecated/svgsfocusable.js

```diff
@@ -0,0 +1,1246 @@
+	},
+	fire: {
+		/* translators: icon label */
+		label: __( 'Fire', 'coblocks' ),
+		keywords: [
+			/* translators: icon keyword */
+			__( 'design', 'coblocks' ),
+			/* translators: icon keyword */
+			__( 'hot', 'coblocks' ),
+			/* translators: icon keyword */
+			__( 'trending', 'coblocks' ),
+			/* translators: icon keyword */
+			__( 'trendy', 'coblocks' ),
+		],
+		icon: <svg height="20" focusable="false" role="img" aria-hidden="true" viewbox="0 0 20 20" width="20" xmlns="http://www.w3.org/2000/svg"><path d="m8.91322329.78180339s.69401426 2.48532134.69401426 4.50171413c0 1.93198565-1.2661071 3.49820703-3.19809275 3.49820703-1.94136422 0-3.40442131-1.56622138-3.40442131-3.49820703l.02813571-.33762856c-1.89447137 2.25085707-3.02927847 5.16759264-3.02927847 8.33754974 0 4.1453284 3.35752846 7.5028569 7.50285689 7.5028569 4.14532848 0 7.50285688-3.3575285 7.50285688-7.5028569 0-5.05504985-2.4290499-9.56614256-6.09607121-12.50163531zm-1.67876423 17.19092081c-1.66938566 0-3.0198999-1.3129999-3.0198999-2.9448713 0-1.5193285.98474997-2.5884856 2.63537848-2.9261142 1.66000709-.3376285 3.37628556-1.1348071 4.33289986-2.41967133.3657643 1.20983563.5533357 2.48532133.5533357 3.78894273 0 2.4853213-2.0163928 4.5017141-4.50171414 4.5017141z" transform="translate(2.553223 -.786621)" /></svg>,
+	},
+	update: {
+		/* translators: icon label */
+		label: __( 'Update', 'coblocks' ),
+		keywords: [
+			/* translators: icon keyword */
+			__( 'add', 'coblocks' ),
+			/* translators: icon keyword */
+			__( 'improve', 'coblocks' ),
+			/* translators: icon keyword */
+			__( 'software', 'coblocks' ),
+			/* translators: icon keyword */
+			__( 'upload', 'coblocks' ),
+		],
+		icon: <svg height="20" focusable="false" role="img" aria-hidden="true" viewbox="0 0 20 20" width="20" xmlns="http://www.w3.org/2000/svg"><path d="m19.9986483 8.20489366h-7.5333334l3.0444445-3.13333333c-3.0333334-3-7.94444447-3.11111111-10.97777781-.11111111-3.03333333 3.01111111-3.03333333 7.86666668 0 10.87777778 3.03333334 3.0111111 7.94444441 3.0111111 10.97777781 0 1.5111111-1.4888889 2.2666666-3.2333333 2.2666666-5.4333333h2.2222223c0 2.2-.9777778 5.0555555-2.9333334 6.9888889-3.9 3.8666666-10.23333331 3.8666666-14.13333331 0-3.88888889-3.8555556-3.92222222-10.12222227-.02222222-13.97777783 3.9-3.85555555 10.15555553-3.85555555 14.05555553 0l3.0333334-3.12222222zm-9.4444445-2.35555555v4.72222219l3.8888889 2.3111111-.8 1.3444445-4.75555555-2.8222222v-5.55555559z" transform="translate(0 -.293783)" /></svg>,
+	},
+};
+
+Object.entries( icons ).filter( function( item ) {
+	svgs.outlined[ item[ 0 ] ] = {
```

Comment:
@AnthonyLedesma It seems we are always accessing the item[0] here and it makes the code a bit hard to read. I would assign it to a const right before `cons elem = item[0]` and `icons[ item[0] ]` seems to be checked in lots of place I would do the same.

Reply:
Good call here @jonathanbardo. I have gone ahead and made a few changes for readability. Now we are using optional chaining to reduce the ternary operators into `logical or` operators. Also setup `const elem = item[ 0 ]`to reference the looped element and switched to use `.forEach` iteration method because `.filter` expects return values whereas `.forEach` does not.